Dispaly links as list

bashimoha opened the pull request:
this fixes #6449

HarshaNalluru commented:
@bashimoha looks like there are ton of conflicts, can you please resolve them?

deyaaeldeen commented on sdk/videoanalyzer/video-analyzer-edge/README.md:
```diff
@@ -7,7 +7,10 @@ Use the client library for Video Analyzer Edge to:
 
-[Product documentation][doc_product] | [Direct methods][doc_direct_methods] | [Source code][source]
+Key links:
+- [Product documentation][doc_product]
+- [Direct methods][doc_direct_methods] 
+- [Source code][source]
```
@hivyas could you please populate these links and perhaps use our standard ones?

deyaaeldeen replied:
never mind, I just found the links below. However, maybe it is still a good idea to use our standard key links list.

deyaaeldeen commented on sdk/web-pubsub/web-pubsub-express/README.md:
```diff
@@ -9,12 +9,13 @@ Use the express library to:
-[Samples][samples_ref]
+  
+Key links:
+- [Source code](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/web-pubsub/web-pubsub-express)
```
```suggestion
- [Source code](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/web-pubsub/web-pubsub-express)
```

deyaaeldeen commented on sdk/web-pubsub/web-pubsub/README.md:
```diff
@@ -25,11 +25,12 @@ Use the library to:
-[Product documentation](https://aka.ms/awps/doc) |
-[Samples][samples_ref]
+Key links:
+- [Source code](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/web-pubsub/web-pubsub)
```
```suggestion
- [Source code](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/web-pubsub/web-pubsub)
```

deyaaeldeen commented on sdk/textanalytics/ai-text-analytics/README.md:
```diff
@@ -21,11 +21,12 @@ Use the client library to:
-[Product documentation](https://docs.microsoft.com/azure/cognitive-services/text-analytics/) |
-[Samples](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/textanalytics/ai-text-analytics/samples)
+Key links:
+- [Source code](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/textanalytics/ai-text-analytics/)
```
```suggestion
- [Source code](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/textanalytics/ai-text-analytics/)
```

deyaaeldeen commented on sdk/tables/data-tables/README.md:
```diff
@@ -16,7 +16,12 @@ Azure Cosmos DB provides a Table API for applications that are written for Azure
 
-[Source code](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/tables/data-tables/) | [Package (NPM)](https://www.npmjs.com/package/@azure/data-tables) | [API reference documentation](https://docs.microsoft.com/javascript/api/@azure/data-tables) | [Product documentation](https://docs.microsoft.com/azure/storage/tables/table-storage-overview/) | [Samples](https://github.com/Azure/azure-sdk- ...
+Key links:
+- [Source code](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/tables/data-tables/)
```
```suggestion
- [Source code](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/tables/data-tables/)
```

deyaaeldeen commented on sdk/storage/storage-queue/README.md:
```diff
@@ -10,12 +10,13 @@ Use the client libraries in this package to:
-[Samples](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/storage/storage-queue/samples) |
-[Azure Storage Queue REST APIs](https://docs.microsoft.com/rest/api/storageservices/queue-service-rest-api)
+Key links:
+- [Source code](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-queue)
```
```suggestion
- [Source code](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/storage/storage-queue)
```

deyaaeldeen commented on sdk/storage/storage-queue/README.md:
```diff
@@ -10,12 +10,13 @@ Use the client libraries in this package to:
+- [Package (npm)](https://www.npmjs.com/package/@azure/storage-queue)
+- [API Reference Documentation](https://docs.microsoft.com/javascript/api/@azure/storage-queue)
+- [Product documentation](https://docs.microsoft.com/azure/storage/queues/storage-queues-introduction)
+- [Samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-queue/samples)
```
```suggestion
- [Samples](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/storage/storage-queue/samples)
```

deyaaeldeen reviewed (commented):
Please revert the master change back to main, I pointed to some of the places that had this issue but it looks like all links need to be revisited.

bashimoha commented:
> Please revert the master change back to main, I pointed to some of the places that had this issue but it looks like all links need to be revisited.
sorry I am having trouble understanding "revert the master change back to main" do you mean I should change the word from master to main? if so then the URL is changing itself  to main when I paste the link to the url bar

deyaaeldeen commented:
> if so then the URL is changing itself to main when I paste the link to the url bar

Yes, but why do you want to rely on this behavior? why changing the word main to master again? we removed the word master for a reason so we do not want to see it again in our READMEs. Thanks.

bashimoha commented:
> > if so then the URL is changing itself to main when I paste the link to the url bar
> 
> Yes, but why do you want to rely on this behavior? why changing the word main to master again? we removed the word master for a reason so we do not want to see it again in our READMEs. Thanks.

I see what you mean. I didn't have idea that I changed it, maybe when I was trying to resolve the conflict? I will try to change it the way it was.

bashimoha commented:
> > if so then the URL is changing itself to main when I paste the link to the url bar
> 
> Yes, but why do you want to rely on this behavior? why changing the word main to master again? we removed the word master for a reason so we do not want to see it again in our READMEs. Thanks.
 
I try to changed it to the way it was, are those okay?

deyaaeldeen reviewed (approved):
Looks good, I left one minor comment

deyaaeldeen commented:
@bashimoha Thanks! I merged the PR.
